helios-client/src/main/java/com/spotify/helios/common/descriptors/RollingOperationTasks.java

```diff
@@ -26,32 +26,32 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
We're not very consistent with this at the moment. My implementation follows what pattern I can see - i.e. this wasn't used in `DeploymentGroupTasks` either (but is in `DeploymentGroup`).

FWIW I'm not a fan of the concept in general. I prefer a JSON API with `nil` values than a JSON API that returns different keys under different circumstances.